api-client/src/main/java/com/projectname/api/client/data/model/users/create/CreateBooksResponse.java

```diff
@@ -0,0 +1,103 @@
+package com.projectname.api.client.data.model.users.create;
+
+import java.io.Serializable;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import com.projectname.api.client.data.model.users.update.UpdateUserRequest;
+import com.projectname.api.client.data.model.users.update.UpdateUserResponse;
+
+public class CreateBooksResponse implements Serializable
```

Comment:
Ovo ti je isti response kao GetBooksByIdResponse, to si mogao da napravis neki CommonBookResponse i da koristis svuda